src/main/java/org/folio/marc/migrations/services/batch/saving/SavingRecordsChunkProcessor.java

```diff
@@ -33,7 +35,10 @@ public DataSavingResult process(OperationChunk chunk) {
     var chunkStep = createChunkStep(chunk);
     var recordsSavingData = new RecordsSavingData(chunk.getOperationId(), chunk.getId(), chunkStep.getId(),
         chunk.getNumOfRecords());
-    var saveResponse = authorityStorageService.saveAuthorities(chunk.getEntityChunkFileName());
+
+    var operationType = operationJdbcService.getOperation(chunk.getOperationId().toString()).getEntityType();
```

Comment:
this db call can be removed, because entityType now exist in JobParameters, and you can get it in this way ```
@Value("#{jobParameters['entityType']}")
  private EntityType entityType;
```

Reply:
Done, thank you!